tools/flaky-test-reporter/gke_deployment/retryer_service.yaml

```diff
@@ -0,0 +1,49 @@
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: flaky-test-retry-bot
+spec:
+  template:
+    metadata:
+      labels:
+        app: flaky-test-retry-bot
+    spec:
+      containers:
+        image: gcr.io/knative-tests/flaky-test-retryer:latest
+        imagePullPolicy: Always
+        command: "/flaky-test-retryer"
+        args:
+        - "--service-account=/etc/google-app-credential/knative-monitoring-credential.json"
+        - "--github-account=/etc/flaky-test-reporter-github-token/token"
+        - "--dry-run=true"
```

Comment:
should this be true or we are testing this first and then update?

Reply:
testing first, I'd like to see how it behaves over the course of a day or two before deploying for real